lib/features/local_live_log.js

```diff
@@ -35,6 +35,9 @@ export default class TaskclusterLogs {
     let taskId = task.status.taskId;
     let runId = task.runId;
 
+    let artifactName = task.payload.log || ARTIFACT_NAME;
+    // let artifactName =  || BACKING_ARTIFACT_NAME
+
     // ensure we have a bulk log backing stuff...
     this.bulkLog = new BulkLog(BACKING_ARTIFACT_NAME);
```

Comment:
Ok, this is also going to need changing.  What happens in the life of the log is:

1. Live log reference artifact is created that points to the container streaming the live log.  This is called a reference/redirect artifact. (this allows someone to view the realtime stream of the log directly from the host.
2. The "bulk log" is created, which is just a temporary file locally first that is the entire log
3. Once the task is complete, the bulk log temporary file is uploaded as an artifact, typically with the name "live_backing.log"
4. the live log redirect artifact is then pointed to that bulk log artifact rather than the container (since the container serving up the live log is going to be gone after the task completes)

If there is a custom name for the live log, such as "/private/jlorenzo/task.log" , then the bulk log must also be stored under this path.  Probably can just do a path.parse(task.payload.log) and toss the backing log under that path when it uploads the artifact.  Hopefully I didn't confuse things.